sui/src/bin/validator.rs

```diff
@@ -57,47 +50,22 @@ async fn main() -> Result<(), anyhow::Error> {
 
     let network_config_path = sui_config_dir()?.join(SUI_NETWORK_CONFIG);
 
-    let network_config = match (network_config_path.exists(), cfg.force_genesis) {
-        (true, false) => PersistedConfig::<NetworkConfig>::read(&network_config_path)?,
+    let validator_config = match (network_config_path.exists(), cfg.force_genesis) {
+        (true, false) => PersistedConfig::<ValidatorConfig>::read(&network_config_path)?,
 
         // If network.conf is missing, or if --force-genesis is true, we run genesis.
         _ => {
-            let genesis_conf: GenesisConfig = PersistedConfig::read(&cfg.genesis_config_path)?;
-            let adddress = SuiAddress::from(genesis_conf.key_pair.public_key_bytes());
-            let (network_config, _, _) = genesis(genesis_conf, Some(adddress)).await?;
-            network_config
+            let mut genesis_conf: GenesisConfig = PersistedConfig::read(&cfg.genesis_config_path)?;
+            genesis_conf.quorum_size = 1;
```

Comment:
This is specifically for the case where a config isn't provided and we want to spin up a lonely validator. I'm not so sure i agree with the overall flow in this case but didn't want to change even more things in this PR since its already long